conans/client/api/conan_api.py

```diff
@@ -348,23 +360,34 @@ def get_package_ids(self, reference, remote=None):
                         "d5f16437dd4989cc688211b95c24525589acaafd": {
                             "settings": {"compiler": "apple-clang",...},
                             "options": {'options': {'shared': 'False',...}},
-                            "requires": ['mylib/1.0.8:3df6ebb8a308d309e882b21988fd9ea103560e16',...]
+                            "requires": ['mylib/1.0.8#3df6ebb8a308d309e882b21988fd9ea103560e16',...]
```

Comment:
This seems too long to be a revision, looks a package_id.

Reply:
My bad, I did not want to change this one 🤦